salt/tests/unit/modules/files/test_metalk8s_monitoring.yaml

```diff
@@ -1,9 +1,14 @@
 alertmanager_api_helper:
   - _id: nominal
-    endpoint: &alertmanager_endpoint
-      ip: '10.0.0.1'
-      ports:
-        web: 1234
+    endpoints: &alertmanager_endpoints
+      - ip: '10.0.0.1'
+        ports:
+          http-web: 1234
+          reloader-web: 1235
+      - ip: '10.0.0.2'
+        ports:
+          web: 2345
+          reloader-web: 2346
```

Comment:
This one with `web` is no longer valid, you should remove it here

Reply:
Removed.